crates/bevy_ui/src/entity.rs

```diff
@@ -136,22 +134,50 @@ impl Default for ButtonBundle {
         }
     }
 }
-#[derive(Component, Clone)]
-pub struct CameraUi {
-    pub is_enabled: bool,
```

Comment:
The meaning of `CameraUi` changed in #4745:
* before: A marker component used to specify which camera was the one for rendering UI
* after: A component to specify whether the camera it is attached to displays the UI (when not present, shows UI)

So technically still used, but for something almost opposite of what it used to.

I renamed it `CameraUiConfig` in #5234, because if you were previously relying on `CameraUi`, your game would still compile, but really do the opposite of what it was doing before.

I built on `CameraUiConfig` in this PR, because I thought it made sense as an API for manipulating the UI camera associated with actual 2d or 3d camera.